.github/workflows/library-release.yml

```diff
@@ -15,9 +15,7 @@ jobs:
       matrix:
         include:
           - os: macos-latest
-            target: aarch64-apple-darwin
-          - os: macos-latest
```

Comment:
(Sidenote: Some people like to separate those out explicitly, and pull in only the compiled code for their specific arch)

Reply:
I re-added the individual Mac targets, so in the release we’ll have all three: ARM, Intel and Universal.